Src/Fido2.Development/StoredCredential.cs

```diff
@@ -52,7 +52,7 @@ public class StoredCredential
 
     public byte[] UserId { get; set; }
 
-    public PublicKeyCredentialDescriptor Descriptor { get; set; }
+    public PublicKeyCredentialDescriptor Descriptor => new(PublicKeyCredentialType.PublicKey, Id, Transports);
```

Comment:
For less allocations we could cache the instance of the PublicKeyCredentialDescriptor as a private member.

Reply:
This looks good as-is to me. The consumer is likely to capture this into a variable, and it's a tiny object.